packages/shared/lib/src/sdk/args.rs

```diff
@@ -534,11 +531,17 @@ pub fn ibc_transfer_tx_args(
     let amount = InputAmount::Unvalidated(denom_amount);
     let port_id = PortId::from_str(&port_id).expect("Port id to be valid");
     let channel_id = ChannelId::from_str(&channel_id).expect("Channel id to be valid");
+    let ibc_shielding_data = match shielding_data {
+        Some(v) => Some(IbcShieldingData::try_from_slice(&v)?),
+        None => None,
+    };
+
```

Comment:
See https://github.com/anoma/namada/blob/9b8252e6c3dd73678c00495b474bb7795a77b6c6/crates/ibc/src/msg.rs#L130-L137

We will need to implement something for `MaspTransaction` for shielding over IBC. Our schema can just handle the Borsh-serialized bytes, but that will need to be constructed in the interface, this just gets or schema up-to-date for IBC